include/deal.II/base/types.h

```diff
@@ -115,7 +119,7 @@ namespace types
 #ifdef DEAL_II_WITH_64BIT_INDICES
   using global_cell_index = std::uint64_t;
 #else
-  using global_cell_index = unsigned int;
+  using global_cell_index       = unsigned int;
```

Comment:
I'm surprised this passes the indentation check.

Reply:
It was the result of clang-format on my machine and I ran make indent as well.
It somehow felt like the `=` on that line and `= long long` should be aligned...

Reply:
Yeah, I don't think this is something to worry about.

Reply:
I think partially it is, because it seems that make indent ignores one of the two definition cases
and maybe it should run for both options somehow.
Otherwise everything would be aligned with
`using signed_global_dof_index =`